tests/app/main/test_suppliers.py

```diff
@@ -2988,31 +2983,31 @@ def test_add_registration_number_page_loads(self):
                  'companies_house_number': '',
                  'other_company_registration_number': ''
                  },
-                'Are you registered with Companies House?',
+                'You need to answer this question.',
```

Comment:
I'm confused, why has this error message text changed?

Reply:
This is no different to any of the other radio button questions - the error message in the top summary is now the instructions for how to solve the problem rather than the question name. It's this question for reference: 

![image](https://user-images.githubusercontent.com/6362602/108076378-449ac900-7063-11eb-9faf-c9f802e288f1.png)

Reply:
Ah, of course. I think we need to revise the error message content, I've added a ticket to do this later https://trello.com/c/AnpNWBC7/285-update-error-messages-for-create-a-supplier-account-journey

Reply:
Yeah I agree. If we update them for the declaration questions as well we can revert #1343 and get rid of even more custom error logic. I'll put a comment on that card.

Reply:
The declaration question error message content is a whole other bag of worms unfortunately 😅 I tried to update them in the past, but it became cumbersome.

Reply:
😢

Reply:
For this page (`/suppliers/registration-number/edit`) the error summary does not link to the first radio button as it should do

Reply:
Good spot, thanks

Reply:
It should be straightforward to fix

Reply:
Should be fixed in c7bbcd7

Reply:
Oh, the page with the "Do. you have a companies house number?" question still isn't working, I think we also need to add `id` to `has_companies_house_number`

https://github.com/alphagov/digitalmarketplace-supplier-frontend/blob/master/app/main/forms/suppliers.py#L92

Reply:
Yup, adding IDs to all three fields seems to have sorted it